canvas_sdk/v1/data/allergy_intolerance.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+from canvas_sdk.v1.data.base import CommittableModelManager
+from canvas_sdk.v1.data.patient import Patient
+
+
+class AllergyIntolerance(models.Model):
```

Comment:
@aduane See the commented out field names below. I mostly followed what you did for Condition, but if there's more data here that we want to expose right now, let me know.

Reply:
We should bring all of those fields over. It's fine if the foreign keys are just integers right now and not full relations. We can work on those later, but fields like:
```
    # allergy_intolerance_type
    # category
    # status
    # severity
    # onset_date_original_input
    # last_occurence_original_input
    # narrative
```

Are all important context.